trillian/log_server/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM scratch
+ADD main /
+VOLUME ["/config"]
+ENV DB_USER root
+ENV DB_PASS=
+ENV DB_HOST db
+ENV DB_PORT 3306
```

Comment:
What do you mean? Honestly, those ENVs don't do anything; because we are using `FROM scratch`, we cannot use shell environment variable expansion when starting the server. That is why I mention wanting to write a trampoline that will load arguments from a config file and then start the actual server.

The port does not need to be exposed to the host machine.